### DISCUSSION TITLE
Can I add a my own column to the quartz table?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2271] [Q&A]
I would like an add a column to one of the quartz table qrtz_triggers, can I add it?

and if yes then how can I insert value to it whenever a new job is scheduled.

this column value will never change for that particular record.

im trying avoiding to avoid the job_group and trigger_group for this purpose as I’m already using them for some other purpose.

**Comment by viveknuna** [comment]
@lahma do you have input on this?

**Comment by lahma** [comment]
Modifying Quartz's tables outside of Quartz is not supported, whether it's CRUD or schema alteration.
